es/refresher-linear-algebra.md

```diff
@@ -126,49 +126,49 @@
 
 **22. Transpose ― The transpose of a matrix A∈Rm×n, noted AT, is such that its entries are flipped:**
 
-&#10230; Transpuesta ― La transpuesta de la matriz A∈Rm×n, con notación AT, es tal que sus entradas son volteadas:
+&#10230; Transpuesta ― La transpuesta de una matriz A∈Rm×n, denotada AT, está dada por el siguiente cambio de posición de sus entradas:
 
 <br>
 
 **23. Remark: for matrices A,B, we have (AB)T=BTAT**
 
-&#10230; Nota: para matrices A,B, se tiene (AB)T=BTAT
+&#10230; Observación: dadas las matrices A,B, se cumple que (AB)T=BTAT
 
 <br>
 
 **24. Inverse ― The inverse of an invertible square matrix A is noted A−1 and is the only matrix such that:**
 
-&#10230; Inversa ― La inversa de una matriz cuadrada invertible A, llamada A−1 y es la única matriz tal que:
+&#10230; Inversa ― La inversa de una matriz cuadrada invertible A es denotada como A−1 y es la única matriz tal que:
 
 <br>
 
-**25. Remark: not all square matrices are invertible. Also, for matrices A,B, we have (AB)−1=B−1A−1**
+**25. Observación: not all square matrices are invertible. Also, for matrices A,B, we have (AB)−1=B−1A−1**
 
-&#10230; Nota: no todas las matrices se pueden invertir. Además, para las matrices A,B, se tiene que (AB)−1=B−1A−1
+&#10230; Nota: no todas las matrices cuadradas son invertibles. Además, para las matrices A,B, se cumple que (AB)−1=B−1A−1
 
 <br>
 
 **26. Trace ― The trace of a square matrix A, noted tr(A), is the sum of its diagonal entries:**
 
-&#10230; Traza ― La traza de una matriz cuadrada A, tr(A), es la suma de sus elementos en la diagonal:
+&#10230; Traza ― La traza de una matriz cuadrada A, denotada tr(A), es la suma de los elementos en su diagonal:
 
 <br>
 
 **27. Remark: for matrices A,B, we have tr(AT)=tr(A) and tr(AB)=tr(BA)**
 
-&#10230; Nota: para matrices A,B, se tiene tr(AT)=tr(A) y tr(AB)=tr(BA)
+&#10230; Observación: dadas las matrices A,B, se cumple que tr(AT)=tr(A) y tr(AB)=tr(BA)
 
 <br>
 
 **28. Determinant ― The determinant of a square matrix A∈Rn×n, noted |A| or det(A) is expressed recursively in terms of A∖i,∖j, which is the matrix A without its ith row and jth column, as follows:**
 
-&#10230; Determinanate ― El determinante de una matriz cuadrada A∈Rn×n, llamado |A| or det(A) es expresado recursivamente en términos de A∖i,∖j, que es la matriz A en su i-ésima fila y j-ésima columna, como se muestra:
+&#10230; Determinante ― El determinante de una matriz cuadrada A∈Rn×n, denotado |A| or det(A), es expresado recursivamente en términos de A∖i,∖j, que es la matriz A sin su i-ésima fila ni su j-ésima columna, de la siguiente forma:
 
 <br>
 
 **29. Remark: A is invertible if and only if |A|≠0. Also, |AB|=|A||B| and |AT|=|A|.**
 
-&#10230; Nota: A es tiene inversa si y solo si |A|≠0. Además, |AB|=|A||B| y |AT|=|A|.
+&#10230; Observación: A es inversible si y solo si |A|≠0. Además, |AB|=|A||B| y |AT|=|A|.
```

Comment:
"Inversible" does not exist. The correct word is "_[invertible](http://dle.rae.es/?id=M3U7pqj)_".